test/grid_animation.jl

```diff
@@ -0,0 +1,158 @@
+using Javis
+using Luxor
+
+function grid_animation()
+
+    video = Video(500, 500)
+    javis(
+        video,
+        [
+            Action(1:100, ground),
+            Action(1:100, :line, (args...) -> draw_grid(args..., grid_direction = :TL)),
+            Action(1:100, :line, (args...) -> zero_lines(args..., line_direction = :TL)),
+        ],
+        tempdirectory = "/home/src/Projects/javis/test/grid_test",
+        creategif = true,
+        pathname = "/home/src/Projects/javis/test/grid_test.gif",
+    )
+
+end
+
+function draw_grid(video::Video, action::Action, frame::Int; grid_direction::Symbol = :TR)
+
+    min_width = video.width / -2
+    max_width = video.width / 2
+
+    min_height = video.height / -2
+    max_height = video.height / 2
+
+    if grid_direction == :BR
+
+        for y_point = min_width:25:max_width
```

Comment:
The 25 should be a variable I guess

Reply:
Definitely agree. Will work on this either now or later in another PR.

Reply:
Now is preferred as also mentioned in my next review comments.